webpack/storybook/atoms/AlertCard.stories.js

```
@@ -0,0 +1,181 @@
+  variant: "success",
+  label: "Success",
+};
+
+export const Warning = Template.bind({});
+Warning.args = {
+  ...commonArgs,
+  label: "Warning",
+  variant: "warning",
+};
+
+export const Error = Template.bind({});
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)